web/client/components/catalog/Catalog.jsx

```diff
@@ -208,6 +219,13 @@ class Catalog extends React.Component {
     renderRecords = () => {
         return (<div className="catalog-results">
                 <RecordGrid {...this.props.gridOptions} key="records"
+                    clearModal={this.props.clearModal}
```

Comment:
All these callbacks are not needed if you put the logic outside the component (e.g. in an epic or an enhancer)